guides/common/modules/proc_adding-proxmox-images-to-project-by-using-cli.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="adding-proxmox-images-to-{project-context}-by-using-cli"]
+= Adding Proxmox images to {Project} by using Hammer CLI
+
+Proxmox uses templates as images for creating virtual machines.
```

Comment:
The intro sentence sounds strange to me, but I don't have a suggestion because I don't know how Proxmox works.